content/cli/commands/completion.md

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
This is from https://github.com/infinyon/fluvio-website/commit/356c92144d84ab4ce6b96be06f28f83b135989b9 which should be omitted.

Reply:
Why should that be omitted?

Reply:
it's not in current release